### DISCUSSION TITLE
Passing variable metadata in Json file uploader

### DISCUSSION THREAD

**Discussion by segelmache** [FlowiseAI/Flowise#1027] [Q&A]
Hey, I need help: Is it possible to define or pass variable (not only static) metadata with the json file uploader? For example I have a pointer named "author". The Author varies per entry in the json file. That would be very helpful! Thanks

**Comment by HenryHengZJ** [comment]
you can pass it in the API using `overrideConfig` body param

**Reply by segelmache** [reply]
@HenryHengZJ  Thank you, is it also possible directly in the no code interface? Like with configuring a node? (e.g. the Json Lines File document loader node; Text Splitter node;...) Thank you.
